acceptance/framework/consul/helm_cluster.go

```diff
@@ -211,12 +211,38 @@ func (h *HelmCluster) Upgrade(t *testing.T, helmValues map[string]string) {
 	k8s.WaitForAllPodsToBeReady(t, h.kubernetesClient, h.helmOptions.KubectlOptions.Namespace, fmt.Sprintf("release=%s", h.releaseName))
 }
 
-func (h *HelmCluster) SetupConsulClient(t *testing.T, secure bool) *api.Client {
+func (h *HelmCluster) CreatePortForwardTunnel(t *testing.T, remotePort int) string {
```

Comment:
Extracted this out so that we can set up a tunnel without having to get another client.

Reply:
Agreed!